homeassistant/components/livisi/coordinator.py

```diff
@@ -92,6 +94,39 @@ async def async_get_pss_state(self, capability: str) -> bool | None:
         on_state = response["onState"]
         return on_state["value"]
 
+    async def async_get_vrcc_target_temperature(self, capability: str) -> float | None:
+        """Get the target temperature of the climate device."""
+        response: dict[str, Any] = await self.aiolivisi.async_get_device_state(
+            capability[1:]
+        )
+        if response is None:
+            return None
```

Comment:
The type hint here is wrong, or you wouldn't need to check for `None`.
(same for below checks)

Reply:
I changed this, thank you.